types/merge.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+import (
+	"github.com/imdario/mergo"
+)
+
+func MergeDerekRepoConfigs(localConfig, remoteConfig DerekRepoConfig) (DerekRepoConfig, error) {
+
+	mergeErr := mergo.Merge(&remoteConfig, &localConfig, mergo.WithAppendSlice)
```

Comment:
Are we going to have the same issues we have with ofc-bootstrap? I saw some tests, do we need more?